src/plugins/data/public/query/query_string/language_service/lib/language_reference.test.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { render, screen, fireEvent, cleanup, act, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
Guess you don't need to import this? Because `@testing-library/jest-dom` is imported in jest setup files.